.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: java
 jdk:
 - openjdk8
-- openjdk11
```

Comment:
is this one failing?

Reply:
No, it's not. This was removed by mistake. Look away! 
This PR is useless as well. Deleting the branch. 😄